public/css/game.scss

```diff
@@ -0,0 +1,185 @@
+.container {
+  width: 100%;
+  margin: 0 !important;
+}
+
+.game-cards {
+  height: 180px;
+  border: 1px solid black;
+  width: 130px;
+  margin: 50px;
+  background-color: black;
+  border-radius: 5px;
+  color: white;
+  p {
+    font-size: smaller;
+  }
+}
+
+#players-cards {
+  height: 500px;
+}
+
+#deck-of-black-cards {
+  box-shadow: -2px -2px 0px 0px  #000000;
+  border-top: 1px solid #ebebe0;
+  border-left: 1px solid #ebebe0;
+  line-height: 180px;
+  text-align: center;
+  font-size: 25px;
+}
+
+#black-card {
+  padding: 10px;
+  height: 200px;
+  width: 150px;
+  font-size: medium;
+  u {
+    position: relative;
+    content: '';
+    display: inline-block;
+    width: 25%;
+    text-decoration: none;
+    height: 1em;
+    border-bottom: 0.125em solid white;
+    margin-bottom: -0.125em;
+  }
+}
+
+#start-game, #awaiting-players {
+  margin-top: 100px;
+}
+
+#start-game p {
+  margin-left: 10px;
+}
+
+#question-holder {
+  padding-left: 100px;
+}
+
+#black-cards-bar {
+  height: 300px;
+  border-bottom: 3px solid #eee;
+}
+
+#white-cards-bar {
+  border-left: 3px solid #eee;
+}
+
+#players-bar {}
+
+#exit {
+  font-size: 18px;
+}
+
+#timer-place {
+  margin-top: 69px;
+}
+
+#timer {
+  padding: 0 10%;
+  font-weight: bolder;
+  font-size: 72px;
+}
+
+#avatar {
+  border-radius: 100%;
+  width: 50px;
+  height: 50px;
+  line-height: 50px;
+  background-color: #4074f5;
+  color: white;
```

Comment:
@demiladebamgbose @andela-oolutola use variables or just write the hex code not color names. You may want to do something in the line of `$white: #ffffff` and use the variable `$white` in all places where you'd like to make reference to the color.